src/hooks/useKeepAlive.tsx

```diff
@@ -1,8 +1,7 @@
 import { useEffect, useState } from 'react'
-import { singletonHook } from 'react-singleton-hook'
```

Comment:
Also used in a single place here so seems sane to me

Reply:
See https://github.com/shapeshift/mobile-app/pull/109#discussion_r2078333976